pkg/template/sample/report/BUILD

```diff
@@ -0,0 +1,51 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
+load("@org_pubref_rules_protobuf//go:rules.bzl", "go_proto_library", "go_proto_compile")
+
+go_proto_compile(
+    name = "go_compiled_src",
+    importmap = {
+        "google/protobuf/descriptor.proto": "github.com/golang/protobuf/protoc-gen-go/descriptor",
+        "google/protobuf/duration.proto": "github.com/golang/protobuf/ptypes/duration",
+        "mixer/v1/config/descriptor/value_type.proto": "istio.io/api/mixer/v1/config/descriptor",
+        "google/protobuf/descriptor.proto": "github.com/golang/protobuf/protoc-gen-go/descriptor",
+        "google/protobuf/struct.proto": "github.com/golang/protobuf/ptypes/struct",
+        "pkg/template/mixer/TemplateExtensions.proto": "istio.io/mixer/pkg/template/mixer",
+    },
+    imports = [
+        "external/com_github_google_protobuf/src",
+        "external/com_github_istio_api",
+    ],
+    inputs = [
+        "//pkg/template/mixer:protos",
+        "@com_github_google_protobuf//:well_known_protos",
+        "@com_github_istio_api//:mixer/v1/config/descriptor_protos",
+    ],
+    protos = ["ReportTesterTemplate.gen.proto"],
+)
+
+genrule(
```

Comment:
I think all of this will go away after you PR is in..  if you are okay, I think we can leave this and after your PR is in, I can replace this with your build rule